ros/src/fsds_ros_bridge/src/airsim_ros_wrapper.cpp

```diff
@@ -161,13 +173,19 @@ void AirsimROSWrapper::create_ros_pubs_from_settings_json()
                     }
 
                     image_pub_vec_.push_back(image_transporter.advertise(curr_vehicle_name + "/" + curr_camera_name + "/" + image_type_int_to_string_map_.at(capture_setting.image_type), 1));
-                    cam_info_pub_vec_.push_back(nh_private_.advertise<sensor_msgs::CameraInfo> (curr_vehicle_name + "/" + curr_camera_name + "/" + image_type_int_to_string_map_.at(capture_setting.image_type) + "/camera_info", 10));
+                    cam_info_pub_vec_.push_back(nh_private_.advertise<sensor_msgs::CameraInfo>(curr_vehicle_name + "/" + curr_camera_name + "/" + image_type_int_to_string_map_.at(capture_setting.image_type) + "/camera_info", 10));
                     camera_info_msg_vec_.push_back(generate_cam_info(curr_camera_name, camera_setting, capture_setting));
+                    // Fill statistics vector here as well using curr_camera_name
+                    ros_bridge::Statistics camera_pub_statistics(curr_camera_name + "_Publisher");
+                    ros_bridge::Statistics camera_rpc_statistics(curr_camera_name + "_RpcCaller");
+                    cam_pub_vec_statistics.push_back(camera_pub_statistics);
+                    simGetImagesVecStatistics.push_back(camera_rpc_statistics);

```

Comment:
this similar procedure is repeated for IMU and LiDAR (which can also have multiple sensors and thus need multiple statistics instances)